README.md

```diff
@@ -2,7 +2,7 @@
 
 # Rubberband
 
-A flexible LP solver log file web view and analysis tool, backed by Elasticsearch.
+A flexible LP solver log file web view and analysis tool, backed by Elasticsearch!
```

Comment:
A flexible web view and analysis platform for solver log files of mathematical optimization software, backed by Elasticsearch!

Reply:
backed by Elasticsearch.